SmalltalkCI class>>saveAndQuitImage currently broken in  Pharo-11.0.0+build.725

`self platformClass saveImage` is not returning a boolean so this method fails:

```smalltalk
saveAndQuitImage
	self platformClass saveImage
		"Close image only if it is not resuming."
		ifFalse: [ self platformClass quitImage ] 
```

 Pharo-11.0.0+build.725

<img width="998" alt="Screen Shot 2024-02-15 at 11 37 53" src="https://github.com/hpi-swa/smalltalkCI/assets/475508/99ef1fa1-a8ea-4404-9579-31aef9d84849">

I had a lot of succesful builds in Pharo 11 for several projects so I wonder why it broke now if it was previously working. What moved? Aren't versions used for this pinned?

For Pharo 12:

```
NonBooleanReceiver: proceed for truth.
SnapshotOperation(ProtoObject)>>mustBeBooleanIn:
SnapshotOperation(ProtoObject)>>mustBeBoolean
SmalltalkCI class>>saveAndQuitImage
UndefinedObject>>DoIt
OCReceiverDoItSemanticScope(OCDoItSemanticScope)>>evaluateDoIt:
OpalCompiler>>evaluate
OpalCompiler>>evaluate:
[
	result := Smalltalk compiler evaluate: aStream.
	self hasSessionChanged
		ifFalse: [
			self stdout
				print: result;
				lf ] ] in EvaluateCommandLineHandler>>evaluate: in Block: [...
FullBlockClosure(BlockClosure)>>on:do:
EvaluateCommandLineHandler>>evaluate:
EvaluateCommandLineHandler>>evaluateArguments
EvaluateCommandLineHandler>>activate
EvaluateCommandLineHandler class(CommandLineHandler class)>>activateWith:
[ aCommandLinehandler activateWith: commandLine ] in PharoCommandLineHandler(BasicCommandLineHandler)>>activateSubCommand: in Block: [ aCommandLinehandler activateWith: commandLine ]
FullBlockClosure(BlockClosure)>>on:do:
PharoCommandLineHandler(BasicCommandLineHandler)>>activateSubCommand:
PharoCommandLineHandler(BasicCommandLineHandler)>>handleSubcommand
PharoCommandLineHandler(BasicCommandLineHandler)>>handleArgument:
[ self handleArgument: self firstArgument ] in [ [ self handleArgument: self firstArgument ]
		on: Exit
		do: [ :exit |
			"If the command line is protected by password, we just exit the image because in non-headless mode the handleExit will let the image open. If the password protection is enabled, it is to avoid to let the access to the image."
			self class commandLinePasswordManager hasPasswordSet
				ifTrue: [ Smalltalk snapshot: false andQuit: true ].
			self handleExit: exit ] ] in PharoCommandLineHandler(BasicCommandLineHandler)>>activate in Block: [ self handleArgument: self firstArgument ]
FullBlockClosure(BlockClosure)>>on:do:
[ [ self handleArgument: self firstArgument ]
		on: Exit
		do: [ :exit |
			"If the command line is protected by password, we just exit the image because in non-headless mode the handleExit will let the image open. If the password protection is enabled, it is to avoid to let the access to the image."
			self class commandLinePasswordManager hasPasswordSet
				ifTrue: [ Smalltalk snapshot: false andQuit: true ].
			self handleExit: exit ] ] in PharoCommandLineHandler(BasicCommandLineHandler)>>activate in Block: [ [ self handleArgument: self firstArgument ]...
[self value.
			"IMPORTANT: Do not step over next line of code. See method comments for details"
			Processor terminateRealActive] in FullBlockClosure(BlockClosure)>>newProcess in Block: [self value....
Error with status code 1:
692 run_script /home/runner/.smalltalkCI/helpers.sh

Error: Process completed with exit code 1.
```

--- a/repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo11.class/class/saveImage.st
+++ b/repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo11.class/class/saveImage.st
@@ -1,0 +1,3 @@
+overrides
+saveImage
+	^ super saveImage isImageStarting

--- a/repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo11.class/methodProperties.json
+++ b/repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo11.class/methodProperties.json
@@ -2,6 +2,8 @@
 	"class" : {
 		"getEnv:" : "EstebanLorenzano 7/22/2021 09:54",
 		"isPlatformCompatible" : "EstebanLorenzano 7/22/2021 09:54",
+		"newTravisID" : "EstebanLorenzano 4/13/2022 21:23",
+		"saveImage" : "noha 2/15/2024 23:22",
 		"testRunnerClass" : "EstebanLorenzano 7/22/2021 09:55" },
 	"instance" : {
 		 } }

--- a/repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo12.class/methodProperties.json
+++ b/repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo12.class/methodProperties.json
@@ -1,5 +1,10 @@
 {
 	"class" : {
-		"isPlatformCompatible" : "CyrilFerlicot 4/17/2023 13:59" },
+		"allCategoryNames" : "MaxLeske 9/30/2023 9:20",
+		"classesInCategory:" : "MaxLeske 9/23/2023 10:06",
+		"codeCoverageClass" : "MaxLeske 9/30/2023 9:20",
+		"isPlatformCompatible" : "CyrilFerlicot 4/17/2023 13:59",
+		"setAuthor:" : "MaxLeske 2/9/2024 7:25",
+		"timeToRun:" : "JohanBrichau 1/14/2024 9:46" },
 	"instance" : {
 		 } }

